views/templates/books-show.html

```diff
@@ -0,0 +1,40 @@
+<div class="col-md-10 col-md-offset-1">
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <div class="row">
+        <div class="col-md-3 thumbnail">
+          <img src="{{booksShowCtrl.book.image}}">
+        </div>
+
+        <div class="col-md-9">
+          <ul class="list-group form-inline">
+            <li class="list-group-item">
+              <h4 class="inline-header">Book Title:</h4>
+              <span ng-hide="editing">{{booksShowCtrl.book.title}}</span>
+              <input ng-show="editing" ng-model="booksShowCtrl.book.title" type="text" class="form-control input-md">
+            </li>
+
+            <li class="list-group-item">
+              <h4 class="inline-header">Book Author:</h4>
+              <span ng-hide="editing">{{booksShowCtrl.book.author}}</span>
+              <input ng-show="editing" ng-model="booksShowCtrl.book.author" type="text" class="form-control input-md">
+            </li>
+
+            <li class="list-group-item">
+              <h4 class="inline-header">Book Release Date:</h4>
+              <span ng-hide="editing">{{booksShowCtrl.book.releaseDate}}</span>
+              <input ng-show="editing" ng-model="booksShowCtrl.book.releaseDate" type="text" class="form-control input-md">
+            </li>
+          </ul>
+        </div>
+        <div class='panel-footer'>
+            <div class='panel-footer'>
+              <!-- <button class='btn btn-danger' ng-click="booksShowCtrl.deleteBook(book)">Delete Book</button> -->
+              <button class='btn btn-info' ng-hide="editing" ng-click="editing = true">Edit Book</button>
+              <button class='btn btn-success' ng-show="editing" ng-click="booksShowCtrl.editBook(book); editing = false; booksShowCtrl.goHome()">Save Changes</button>
```

Comment:
Instead of using so many lines in your ng-click, wrap this all inside a single function within your controller.